modules/cloudwatch_dashboard.tf

```diff
@@ -0,0 +1,451 @@
+resource "aws_cloudwatch_dashboard" "main" {
+  count          = var.cloudwatch_dashboard_toggle == "true" ? 1 : 0
+  dashboard_name = "DCE-${var.namespace}"
+
+  dashboard_body = <<EOF
+{
```

Comment:
Can we put this JSON in the `./fixtures` directory or someplace else and then provide it to the resource using the `file()` TF helper? I would like to get rid of the inline usage we have here.